padglobal/padglobal.py

```diff
@@ -311,21 +320,44 @@ def list_or_none(l):
         if not self.c_commands:
             self.c_commands = {}
 
-        op = 'EDITED' if command in self.c_commands else 'ADDED'
+        if text in self.c_commands:
```

Comment:
This seems like a risky and confusing implementation. Why don't you store the aliases separately from the commands instead?

This eliminates the issue I noticed later on where you repeatedly resolve commands. Just make it so that an alias can only point to a root command and not another alias.

Reply:
done

Reply:
This doesn't look done. It doesn't look changed at all.

Reply:
Added safety nets and input from River.  Do you want me to have an entirely new bot_settings dictionary for it?  That sounds like more trouble than it's worth, but I'd be happy to do it

Reply:
It's up to you, but my strong suggestion would be to not store semantically different data types within the same slot in an untyped datastructure. This is a bad choice.

If you're still inclined to store it this way, resolve the discussion, since you're the one maintaining this now ;P